### ISSUE TITLE
Training a model using Torch XLA without TPU Pods/Pod Slices

### ISSUE THREAD

**Issue by Arka161** [pytorch/xla#3153]
## ❓ Questions and Help
Hello, 

For a student project, we were trying to train a model using PyTorch (PyTorch XLA) across multiple TPUs provided by [TPU Research Cloud](https://sites.research.google/trc/about/). The TPUs that we have access to are mentioned in the image below. 

![image](https://user-images.githubusercontent.com/20723780/135940083-180bddea-9590-4279-917d-893b256597a8.png)

Basically, we do not have access to pod slices, and as PyTorch / XLA requires all TPU VMs to be able to access the model code and data, I believe we cannot use the default PyTorch XLA training code the standard way for Multi-Host TPU VMs, and we might have to maneuver `sea of devices` mentioned [here](https://github.com/pytorch/xla/blob/e93451614401d6a53b50ae119dc4ccdceb09a207/torch_xla/core/xla_model.py#L61) or some other distributed strategy for efficient parallelization. 

Is there any recommended way to approach this problem using `Torch.dist` for Multi-Host GCP TPU VMs, or any guide on how we can leverage `sea of devices`  in a simple way? Or, is there any easier technique where we could use `torch.xla` for anything without having a __hackier__ approach to the problem? 

Thanks in advance, 
Appreciate the help!

**Comment by JackCaoG** [comment]
Hi @Arka161, you would not be able to use the `xla_dist` script we usually used for for pt/xla pod training. As of sea of devices mode, I think there are some works to be done on the python API to make it fully functional(pr in [here](https://github.com/pytorch/xla/pull/2314/files)). This feature has been in pause due to original author left and the lack of user request. We also never tried `sea of device` on the TPUVM and I think it will require some work there too.

In short `sea of device` mode is not officially supported and there is not an easy way to leverage it yet. I can add this to our todo list again but it will unlikely to be finished anytime soon.
